/usr/bin/iceauth:  creating new authority file /run/user/1000/ICEauthority

(xfsettingsd:5834): xfsettingsd-WARNING **: 08:11:27.268: Unknown mode '1920x1080 @ 60.0' for output VGA1, aborting.
libpager-Message: 08:11:27.705: Setting the pager rows returned false. Maybe the setting is not applied.

(wrapper-2.0:5912): Gtk-WARNING **: 08:11:27.887: Negative content width -3 (allocation 1, extents 2x2) while allocating gadget (node button, owner GtkToggleButton)

(wrapper-2.0:5914): Gtk-WARNING **: 08:11:27.891: Negative content width -3 (allocation 1, extents 2x2) while allocating gadget (node button, owner GtkToggleButton)

(wrapper-2.0:5910): Gtk-WARNING **: 08:11:27.926: Negative content width -3 (allocation 1, extents 2x2) while allocating gadget (node button, owner PulseaudioButton)
blueman-applet 08.11.28 ERROR    PluginManager:90 load_plugin: Unable to load plugin module AppIndicator
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/AppIndicator.py", line 7, in <module>
    require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/blueman/main/PluginManager.py", line 88, in load_plugin
    importlib.import_module(self.module_path.__name__ + ".%s" % plugin)
  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.7/site-packages/blueman/plugins/applet/AppIndicator.py", line 9, in <module>
    raise ImportError("AppIndicator3 not found")
ImportError: AppIndicator3 not found
blueman-applet 08.11.28 WARNING  PluginManager:147 __load_plugin: Not loading PPPSupport because its conflict has higher priority
blueman-applet 08.11.28 WARNING  PluginManager:147 __load_plugin: Not loading DhcpClient because its conflict has higher priority
libpager-Message: 08:11:51.604: Setting the pager rows returned false. Maybe the setting is not applied.
libpager-Message: 08:11:51.612: Setting the pager rows returned false. Maybe the setting is not applied.
Failed to load English plurals localization.
Directory: /usr/lib/ripcord/translations
File: ripcord_en.qm
qt.network.ssl: QSslSocket: cannot resolve SSL_set_alpn_protos
qt.network.ssl: QSslSocket: cannot resolve SSL_CTX_set_alpn_select_cb
qt.network.ssl: QSslSocket: cannot resolve SSL_get0_alpn_selected

(xfdesktop:5906): libxfce4ui-WARNING **: 08:12:02.702: ICE I/O Error

(xfdesktop:5906): libxfce4ui-WARNING **: 08:12:02.702: Disconnected from session manager.

(xfwm4:5825): libxfce4ui-WARNING **: 08:12:02.702: ICE I/O Error

(xfwm4:5825): libxfce4ui-WARNING **: 08:12:02.702: Disconnected from session manager.
Gdk-Message: 08:12:02.724: xfwm4: Fatal IO error 4 (Interrupted system call) on X server :0.0.

Gdk-Message: 08:12:02.730: xfdesktop: Fatal IO error 4 (Interrupted system call) on X server :0.0.

